ui/components/multichain/network-list-item/network-list-item.stories.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { NetworkListItem } from '.';
+
+export default {
+  title: 'Components/Multichain/NetworkListItem',
+  component: NetworkListItem,
+  argTypes: {
+    name: {
+      control: 'text',
+    },
+    selected: {
+      control: 'boolean',
+    },
+    onClick: {
+      action: 'onClick',
+    },
+    onDeleteClick: {
+      action: 'onDeleteClick',
+    },
+    iconSrc: {
+      action: 'text',
+    },
+  },
+  args: {
+    name: 'Ethereum',
+    iconSrc: '',
+    selected: false,
+    onClick: () => undefined,
+    onDeleteClick: () => undefined,
```

Comment:
Since, we already defined the actions for onClick and onDeleteClick, I think we can remove these from args.

```suggestion
```